src/Exception/ConnectException.php

```diff
@@ -10,15 +12,19 @@
  *
  * Note that no response is present for a ConnectException
  */
-class ConnectException extends RequestException implements NetworkExceptionInterface
+class ConnectException extends TransferException implements NetworkExceptionInterface
```

Comment:
If this gets approved we will need to update the UPGRADE guide and also the changelog.. Added as a note here in order not to forget.

Reply:
I've added a note to _UPGRADING.md_ in 3edf0a8